CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Add empty line in README.md to fix badges. ([#2729](https://github.com/nf-core/tools/pull/2729))
 - Replace automatic branch detection in `nf-core download` CI test with hardcoded `dev` and input. ([#2727](https://github.com/nf-core/tools/pull/2727))
+- Add Github Action to automatically cleanup ubuntu-latest runners to fix runner running out of diskspace errors([#2754](https://github.com/nf-core/tools/issues/2754))
```

Comment:
```suggestion
- Add Github Action to automatically cleanup ubuntu-latest runners to fix runner running out of diskspace errors([#2755](https://github.com/nf-core/tools/issues/2755))
```

Reply:
😱 

issue vs PR, both valid

Reply:
ah, okay, we usually point to the PR, so I would go for that.